web/src/components/Editor/Editor.tsx

```diff
@@ -46,16 +47,24 @@ const Editor = forwardRef((props: Props, ref: React.ForwardedRef<EditorRefAction
       focus: () => {
         editorRef.current?.focus();
       },
-      insertText: (rawText: string) => {
+      insertText: (rawText: string, mark = false) => {
```

Comment:
What about `isMark`?

Reply:
I am ok with that, but after this, we need an "if" to separate the logic.

With prefix and suffix:

```ts
insertText: (content = "", prefix = "", suffix = "") => {
  if (!editorRef.current) {
    return;
  }
  const cursorPosition = editorRef.current.selectionStart;
  const endPosition = editorRef.current.selectionEnd;

  const prevValue = editorRef.current.value;
  const value =
    prevValue.slice(0, cursorPosition) +
    prefix +
    prevValue.slice(cursorPosition, endPosition) +
    suffix +
    content +
    prevValue.slice(endPosition);

  editorRef.current.value = value;
  editorRef.current.focus();
  editorRef.current.selectionEnd = endPosition + prefix.length + suffix.length + content.length;
  handleContentChangeCallback(editorRef.current.value);
  refresh();
},
```

Reply:
Yeah, I thought about that, but it would complicate the `insertText` on the argument passing.
Anyway, it's more intuitive and flexible, I will do it.